.github/CODEOWNERS

```diff
@@ -12,8 +12,11 @@
 /packages/workspace-tools @bigcommerce/team-checkout
 
 ## Shared
-package.json @bigcommerce/team-checkout @bigcommerce/team-integrations @bigcommerce/team-paypal
-package-lock.json @bigcommerce/team-checkout @bigcommerce/team-integrations @bigcommerce/team-paypal
+package.json @bigcommerce/team-checkout @bigcommerce/team-payments @bigcommerce/team-integrations @bigcommerce/team-paypal
+package-lock.json @bigcommerce/team-checkout @bigcommerce/team-payments @bigcommerce/team-integrations @bigcommerce/team-paypal
```

Comment:
Could you please explain why adding `@bigcommerce/team-payments`? What dependencies you are looking to manage in particular?

Reply:
it's not necessary for us. I can remove it 👍